### ISSUE TITLE
Diacritics

### ISSUE THREAD

**Issue by DannyCooper** [google/fonts#1702]
When viewing a font on fonts.google.com all letters appear to display perfectly.

![Example](https://i.stack.imgur.com/qMzqU.png)

But when using them on a site, the diacritics look different to the rest of the font.

@import url('https://fonts.googleapis.com/css?family=Crimson+Text&subset=all');

https://codepen.io/dannycooper147/pen/wEmaoW

Can anyone help me understand why?

**Comment by laerm0** [comment]
Yep, I see this as well. I believe that not all browsers support the `subset` feature; that being said, I tried Safari, Firefox, and Chrome on OS X and it wasn't working in any of those. @davelab6 ?

**Comment by jjlankinen** [comment]
For me diacritics work with every browser except  Chrome in MacOS, several linux distro's and Windows from 7 to 10.

Only Chrome occasionally refuses to display characters with diacritics in fonts.google.com as well as any website. In fonts.google.com characters are left out completely and in websites that uses lets say Open Sans the characters are rendered with fallback font instead and appear a bit thicker if used 300 font weight.

**Comment by sladen** [comment]
@jjlankinen: please can you copy and paste:

1. *exact* snippets that are being used to load + display the font
2. a screenshot + *exact* User-Agent of each browser(s) showing the issue
3. *ideally* a trace of the intermediate CSS and URLs being fetched by the browser (eg. Firebug/Developer tools etc)

This should make it easier to spot patterns/cacheing problems/etc

**Comment by jjlankinen** [comment]
The way we load the font is exactly like it's guided in fonts-google.com

I have tried to load the Open Sans font with 2 different ways:
`@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,600&subset=cyrillic'); `

`<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600&amp;subset=cyrillic" rel="stylesheet">`

Neither of those methods work with Chrome as every character with accent gets replaced by fallback font, Arial or sans-serif in our case. Other tested browsers (FF, Safari, Edge) seems to be rendering diacritics just fine.

Like I said, it only happens in Chrome and it can be seen happening in font-google.com site as well in Open Sans font specimen view when entering following text: Приве́т, Минна! Ты что, то́же е́дешь в Пи́тер. For me characters with accent mark are completely missing (I assume specimen view should display font in its full glory ie. loading every subset browser deems necessary).

Screenshot of said issue in fonts.google.com:
![deepinscreenshot_select-area_20190215101402](https://user-images.githubusercontent.com/3032883/52843414-b2295d00-310a-11e9-982f-c08bf4c85acd.png)

Screenshot of said issue in out website:
![deepinscreenshot_select-area_20190215101644](https://user-images.githubusercontent.com/3032883/52843453-d5eca300-310a-11e9-8e6b-830029c16092.png)

Using Google Chrome Version 72.0.3626.109 (Official Build) (64-bit)

Today issue has been verified again with multiple Windows 10 64bit systems using latest Chrome as well. Other browsers keeps on working just fine.

Can this be Open Sans issue primarily as Crimson Text font works somewhat better in google fonts specimen view but not when added to our website. Open Sans works in google fonts site when manually selecting font and adding weight variations and necessary subsets  (these should be not needed as Chrome supports unicode range right?).

Do you want me to provide screen shot of dev tools network view to show what has been loaded regarding css and fonts?

**Comment by risinek** [comment]
Workaround until this gets fixed is to download to font in TTF format and import it directly in CSS. Then it corresponds with the previews in Google Font app and contains all the special chars. 
**Example:**
```
@font-face {
  font-family: "Special Elite";
  src: url('../font/SpecialElite-Regular.ttf') format('truetype');
}
```
Although this is not recommended as not all browsers supports TTF, it's better than nothing and you can always use fallback fonts (e.g the same font but using Google APIs that will not currently show special chars).
